.prettierignore

```diff
@@ -1,7 +1,8 @@
-node_modules/
-.git/
-coverage/
 .cache/
-lib/
+.nyc_output/
+coverage/
 dist/
-package.json
```

Comment:
package.json should not be formatted with prettier because yarn/npm override the formatting and format differently